cmd/bundle/main_test.go

```diff
@@ -188,14 +211,16 @@ var _ = Describe("Bundle Loader", func() {
 		})
 
 		AfterEach(func() {
-			ref, err := name.ParseReference(testImage)
-			Expect(err).ToNot(HaveOccurred())
+			if testImage != "" {
```

Comment:
When certain tests are skipped, the `testImage` string can actually be empty and therefore would cause a panic.